policy/datasets.yaml

```diff
@@ -779,3 +779,14 @@ fgcmStandardStars: # Catalog of standard stars with magnitudes as generated from
     tables: raw
     python: lsst.afw.table.BaseCatalog
     template: fgcm-process/fgcmStandardStars-%(fgcmcycle)02d.fits
+apPipe_config:
+    persistable: Config
+    python: lsst.ap.pipe.ApPipeConfig
+    storage: ConfigStorage
+    template: config/apPipe.py
+apPipe_metadata:
+    persistable: PropertySet
+    storage: BoostStorage
+    python: lsst.daf.base.PropertySet
+    tables: raw
+    template: ''
```

Comment:
This looks fine. However, it is disappointing that so many obs_ packages must override the default templates. Is there truly no way to provide a more useful default that could be used by more obs packages?